packages/fluentui/react-northstar/src/components/Tooltip/Tooltip.tsx

```diff
@@ -86,6 +86,10 @@ export interface TooltipProps
 
   /** Element to be rendered in-place where the tooltip is defined. */
   trigger?: JSX.Element;
+
+  dismissOnContentMouseEnter?: boolean;
```

Comment:
I am sure this is not the perfect name, accepting suggestions on it 😄 @ling1726 @layershifter